src/components/NavigationBar/index.js

```diff
@@ -187,6 +187,10 @@ class NavigationBar extends Component {
   };
 
   handleSearch = value => {
+    if (typeof value !== 'string') {
```

Comment:
What is this check for?

Reply:
Won't default param value work here?

Reply:
@akshatnitd When onClose function is called it sends an object to the this.handleSearch instead of sending an empty string.